### ISSUE TITLE
single quotes in strings break the pure codegen parser

### ISSUE THREAD

**Issue by josh33901** [stepancheg/rust-protobuf#337]
Example:
```protobuf
import "google/protobuf/descriptor.proto";

extend .google.protobuf.MethodOptions {
	optional string method_description = 50000;
}

message test_message {
    optional string a = 1 [default = "this won't parse"];
}

service test {
    rpc NiceFunction (Map<int32, string>) returns (int32) {
        option (method_description) = "Does nothing and doesn't care";
    }
}
```

and gives error
```
error: ParserErrorWithLocation(ParserErrorWithLocation { error: InternalError, line: 8, col: 48 })
```

using `protobuf-codegen-pure = "2.0.4"`

Am I missing something obvious?

**Comment by stepancheg** [comment]
Published fix as 2.0.5. Thanks!

**Comment by josh33901** [comment]
Thank you!
